components/social_support/ThirdLevelCountDown.tsx

```diff
@@ -0,0 +1,169 @@
+import { Text, StyleSheet, ScrollView, Pressable, View } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import Title from '../Title';
+import { BLACK, DARK_GREY, ORANGE, PRIMARY, SIZES, TERTIARY } from '../../styles';
+import { NavigationProp, useNavigation } from '@react-navigation/native';
+import { SocialSupportStackParamList } from './SocialNavigation';
+import CountDown from 'react-native-countdown-component';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { differenceInSeconds } from 'date-fns';
+
+
+/** source for storage code https://aloukissas.medium.com/how-to-build-a-background-timer-in-expo-react-native-without-ejecting-ea7d67478408 */
+export default function ThirdLevelCountDown() {
+  const { navigate } = useNavigation<NavigationProp<SocialSupportStackParamList>>();
+
+  const timeLimit = 1;
+  // 14 * 24 * 60 * 60
+  const [toggle, setToggle] = useState<boolean>();
+
+  const [toggleFwd, setToggleFwd] = useState(false);
+  const [secondsLeft, setSecondsLeft] = useState<number>();
+
+  /** store the timestamp when the start btn was clicked */
+  async function recordStartTime() {
+    try {
+      const now = new Date();
+      await AsyncStorage.setItem('@start_time', now.toISOString());
+    } catch (err) {
+      console.warn(err);
+    }
+  }
+  /** if sthg stored in async as start time challeng is running => hide start btn */
+  async function setToggeling(): Promise<boolean> {
+    if ((await AsyncStorage.getItem('@start_time')) !== null) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  /** when smth was in async storage then update the timer */
+  const setCountDown = async () => {
+    const startTime = await AsyncStorage.getItem('@start_time');
+    const now = new Date();
+    if (startTime) {
+      const timeLeft = timeLimit - differenceInSeconds(now, Date.parse(startTime));
```

Comment:
Für das Mood Diary habe ich bereits die sehr gute Library js-joda eingebunden. Darin kann man differenceInSeconds ersetzen durch:
```javascript
ChronoUnit.SECONDS.between(Instant.now(), Instant.parse(startTime));
```
oder 
```javascript
Duration.between(Instant.now(), Instant.parse(startTime)).seconds();
```

Dann brauchen wir hier keine neue Library.

(Am liebsten würde ich alle `Date`s durch `Instant`s ersetzen, aber das muss nicht jetzt sein :wink:)

Reply:
Dann für die Zukunft: Ich habe beim satten der Notification einen Datetimepicker verwendet, kann man da auch mit instants arbeiten? :O

Reply:
Und aus Interesse, wieso Instants?

Reply:
Ok, danke für die Erklärung. Hab das in dem TimerLevel jetzt erstmal auf Instant angepasst